### ISSUE TITLE
Cannot convert QueryString to IqueryCollection

### ISSUE THREAD

**Issue by chloecheung** [NonFactors/MVC6.Grid#45]
I am trying to export records by excel. But when I pass a Querystring to grid.Query like you mention in your [documentation](http://mvc6-grid.azurewebsites.net/Grid/Export), it gives me an error saying "Cannot implicitly convert type 'Microsoft.AspNetCore.Http.QueryString' to 'Micorsoft.AspNetCore.Http.IQueryCollection' ". 
Here is the sample code: 
`grid.Query = Request.QueryString;`
Another issue: 
When I comment "grid.Query" out, I got NullReferenceException in grid.Rows. I am not sure what grid.Rows get is right. It gets NonFactor.Mvc.Grid.GridRows<"myModel">. Also CssClass and and currentRows are null. Where does the grid.Rows get these data?  

I am really new to Nonfactor so the questions might be messy. I will try to explain more if it's not clear.

**Comment by Muchiachio** [comment]
Sorry it should be `grid.Query = Request.Query`, will fix that in documentation. It should fix NRE errors too.

**Comment by chloecheung** [comment]
It works. Thanks very much.

**Comment by chloecheung** [comment]
Hi, Muchiachio @Muchiachio 

I just realize I couldn't download any file. Then I try to download excel in your [documentation](http://mvc6-grid.azurewebsites.net/Grid/Export) and it's also not able to download. Still not sure if they are caused by the same issue.

**Comment by Muchiachio** [comment]
Just saw that `Epplus.Core` doesn't support targeting both frameworks for some reason, reverted it back.

**Comment by chloecheung** [comment]
@Muchiachio I am sorry I don't quite understand the frameworks you mentioned.  I couldn't find any changes in sample codes. Would you mind to explain more? Thanks!

**Comment by Muchiachio** [comment]
Frameworks targeted by the documentation project, at the beginning I only targeted full .NET 4.6 framework, which had a dependency on `EPPlus` for creating an excel file.

After some time I found an unofficial port of `EPPlus` to .NET core (a subset .NET framework for cross platform development), so I added that to the documentation project, but I found out that it works only then documentation project target one of those frameworks but not both.

You won't see anything changed in the files, I just changed some project dependencies so `EPPlus` would find all referenced assemblies correctly.
